layouts/partials/scalers.html

```diff
@@ -1,10 +1,15 @@
 {{ $here    := $.Page.RelPermalink }}
 {{ $scalers := where site.RegularPages ".CurrentSection.Title" "Scalers" }}
+{{ $thisVersion    := index (split .File.Path "/") 1 }}
 <div class="buttons has-extra-top-margin">
   {{ range $scalers }}
   {{ $isHere := eq .RelPermalink $here }}
+  {{ $version := index (split .File.Path "\\") 1 }}
```

Comment:
Use the `/` separator here as well, instead of `\\`

Reply:
This works fine on my machine:
`{{ $version := index (split .Dir "\\") 1 }}`

But must be because I'm on Windows.